packages/cli/README.md

```diff
@@ -1 +1,246 @@
-# cli
+# Bucket CLI
+
+Command-line interface for interacting with Bucket services. The CLI allows you to manage apps, features, authentication, and generate TypeScript types for your Bucket features. With this tool, you can streamline your feature flagging workflow directly from your terminal.
+
+## Installation
+
+You can install the Bucket CLI either locally to your project or globally on your system.
+
+### Local Installation (Recommended)
+
+Local installation is recommended for project-specific usage and ensures consistent CLI versions across your team.
+
+```bash
+# npm
+npm install --save-dev @bucketco/cli
+
+# yarn
+yarn add --dev @bucketco/cli
+
+# pnpm
+pnpm add --save-dev @bucketco/cli
+```
+
+When installed locally, use the CLI with npx:
+
+```bash
+# npm
+npx bucket <command>
+```
+
+or as a `package.json` script
+
+```json
+{
+  "scripts": {
+    "bucket": "bucket"
+  }
+}
+```
+
+```bash
+# npm
+npm bucket <command>
+
+# yarn
+yarn bucket <command>
+```
+
+### Global Installation
+
+Global installation makes the CLI available system-wide.
+
+```bash
+# npm
+npm install -g @bucketco/cli
+
+# yarn
+yarn global add @bucketco/cli
+
+# pnpm
+pnpm add -g @bucketco/cli
+```
+
+When installed globally, use the CLI directly:
+
+```bash
+bucket <command>
+```
```

Comment:
feels like we should take this out. Personally my experience with global installations is that it's more pain than it's worth?

Reply:
Demphasised and made it an alternative further down.

Reply:
perfect, thanks!